frontend/src/hooks/useMetricsStepValidationCheckContext.tsx

```diff
@@ -37,14 +43,14 @@ const emptyErrorMessages = {
   steps: '',
 }
 
-const getErrorMessage = (label: string, value: string, id: number, duplicatedPipeLineIds: number[]) =>
+const setErrorMessage = (label: string, value: string, id: number, duplicatedPipeLineIds: number[]) =>
```

Comment:
could this function name  be better？

Reply:
Yes, I have rename this function to assignErrorMessage to decrease misunderstanding